spaceship/lib/spaceship/client.rb

```diff
@@ -829,6 +831,10 @@ def send_request(method, url_or_path, params, headers, &block)
           raise AppleTimeoutError.new, "Apple 302 detected - this might be temporary server error, check https://developer.apple.com/system-status/ to see if there is a known downtime"
         end
 
+        if response.body.to_s.include?("<h3>Bad Gateway</h3>")
```

Comment:
This is only returned with status code 502? Then we should maybe check for that as well (or have an additional `if` for that)?

Reply:
I have no idea. How about checking status codes satisfy 5XX?

```
&& response.status >= 500 && response.status <= 599
```

Reply:
Let's keep the `BadGatewayError` only for 502 responses.